### PR TITLE
Selector enhancement

### PR DESCRIPTION
**Pull request by Niandalu**

### CODE REVIEW

File: src/SelectorWithPanel/Panel/BlockList/AccordionStyle.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      </Collapse>
+    </div>
+  )
+}
+
+const Accordion = (props, context) => {
+  return (
+    <div className="picked-items picked-items__height-breadcrumb">
+      <BsAccordion>
+      {props.dataSource.map((r, i) => <Row eventKey={i} r={r} key={i} onRemove={() => { props.onRemove(r[context.idKey]) }} />)}
```
Comment (sunkai612) [inline src/SelectorWithPanel/Panel/BlockList/AccordionStyle.js]: Do we need to pass in `onSelect` props here?

File: src/utils/Helpers.js [hunk 2]
```diff
@@ -95,5 +95,13 @@ export default {
       newObj[key] = this.filterInt(obj[key], 10) || obj[key];
       return newObj;
     });
+  },
+
+  t(name) {
```
Comment (Xuhao) [inline src/utils/Helpers.js]: How about use another name to different from `t` of I18n?

File: src/SelectorWithPanel/Panel/BlockList/NestedStyle.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React, { PropTypes } from 'react';
+import Row from './Row';
+
+import nodeSchema from '../../Schema/Node';
+
+function build(rows, props) {
```
Comment (Xuhao) [inline src/SelectorWithPanel/Panel/BlockList/NestedStyle.js]: Any reason to not use ES6 syntax  `const build = (rows, props) => {}`  ?

File: src/SelectorWithPanel/Panel/BlockList/AccordionStyle.js (hunk 1 above)
Comment (Niandalu) [reply src/SelectorWithPanel/Panel/BlockList/AccordionStyle.js]: `onSelect` is added by `BsAccordion`, I'm going to add comments later.

File: src/utils/Helpers.js (hunk 2 above)
Comment (Niandalu) [reply src/utils/Helpers.js]: We're using `t('tag name')` as tag helper in DSP. So for consistency concern, `t` is also adopted in `ui-components`. I think it's ok as we're not using i18n shortcut.

File: src/SelectorWithPanel/Panel/BlockList/NestedStyle.js (hunk 3 above)
Comment (Niandalu) [reply src/SelectorWithPanel/Panel/BlockList/NestedStyle.js]: I don't have any specific reason for that ... = = If I have to name one, it should be `() => { () => {} }` is too ugly(maybe). The differences between original style and arrow style are:

* the binding of this
* prototype chain

None of them are being utilized in this case. So either old-style or arrow style should be fine.

**Pull request merged**